packages/web3-eth-personal/src/rpc_method_wrappers.ts

```diff
@@ -50,11 +50,10 @@ export const importRawKey = async (
 	requestManager: EthPersonalAPIManager,
 	keyData: HexString,
 	passphrase: string,
-) => {
-	validator.validate(['bytes', 'string'], [keyData, passphrase]);
+) =>
+	// validator.validate(['bytes', 'string'], [keyData, passphrase]);
```

Comment:
This shouldn't be commented out

Reply:
we have a problem with our validator. need to time to more investigate. but when I pass the correct private key it is not valid. but without validator it works